sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -106,6 +111,25 @@ public DocTableInfo getWritableTable(TableIdent tableIdent) {
             throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
                 "%s is an alias. Write, Drop or Alter operations are not supported", tableInfo.ident()));
         }
+        if (docTableInfo.closed()) {
+            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
+                "The table %s is closed. Write, Drop or Alter operations (other than OPEN) are not supported",
+                tableInfo.ident()));
+        }
+        return docTableInfo;
+    }
+
+    public DocTableInfo getOpenableOrClosableTable(TableIdent tableIdent) {
+        TableInfo tableInfo = getTableInfo(tableIdent);
+        if (!(tableInfo instanceof DocTableInfo)) {
+            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
+                "The table %s is read-only. Write, Drop or Alter operations are not supported", tableInfo.ident()));
```

Comment:
I wonder if we should make these accessors less specific and just have getUserDefinedTable/getSystemTable and implicitly assume that any user-tables are droppable, writeable and closeable